.gitlab-ci.yml

```diff
@@ -168,12 +168,13 @@ snap-bindist:
     expire_in: 1 week
   script:
     # TODO: Clean up nix-store after building
-    - export XZ_DEFAULTS="-T 0 -3"
+    - export THREADS=$(nproc)
+    - export XZ_DEFAULTS="-T ${THREADS} -3"
     - export ROOT=$(pwd)
     - tar xf usr_nix.tar.xz -C / || true
     - apt update
     - apt install ksh -y  # TODO: Remove need for ksh in mkBinDist.sh
-    - cd bindist/linux/snap && ./mkBinDist.sh &> $ROOT/nix_build.log
+    - cd bindist/linux/snap && ./mkBinDist.sh |& tee $ROOT/nix_build.log | egrep -i '^(building|copying)'
```

Comment:
Is this intentional or left over debugging stuff?

Reply:
Originally for debugging, but it's helpful to see progress so I left it in. (Full log is way to large, so gitlab refuses to show it. `nix_build.log` is published as an artifact.)